hack/release-prep.yaml

```diff
@@ -0,0 +1,139 @@
+# This file is used to template updates to version and image tags in the current
+# branch of this repository's source code.
+#
+# The contents of this file are consumed by hack/update-release-parameters.sh,
+# and applied to templates found in hack/release-templates. Double curly braces
+# are used to indicate where replacements should be made in template files, e.g.
+# {{pgmonitor-version}} will be replaced with the value of the pgmonitor-version
+# release tag listed below. The target files to be updated are listed under
+# `updates:` below.
+---
+release-tags:
+  # Versions
+  pgmonitor-version: v4.8.0
+  pgo-latest-version: 5.3.0
+  postgres-15-version: 15.1
+  postgres-14-version: 14.6
+  postgres-13-version: 13.9
+  postgres-12-version: 12.13
+  postgres-11-version: 11.18
+  postgres-exporter-version: 0.10.1
+  k8s-version-range: 1.22-1.25
+  oc-version-range: 4.8-4.11
+
+  # Image tags
+  ubi8-pgbackrest-tag: ubi8-2.41-2
+  ubi8-pgbouncer-tag: ubi8-1.17-5
+  ubi8-pgbouncer-component-tag: ubi8-1.17-1
+  ubi8-pgadmin4-tag: ubi8-4.30-8
+  ubi8-pgupgrade-tag: ubi8-5.3.0-0
+  ubi8-postgres-exporter-tag: ubi8-5.3.0-0
+  ubi8-postgres-15-tag: ubi8-15.1-0
+  ubi8-postgres-14-tag: ubi8-14.6-2
+  ubi8-postgres-14-component-tag: ubi8-14.6-3.1-2
+  ubi8-postgres-13-tag: ubi8-13.9-2
+  ubi8-postgres-gis-14-32-tag: ubi8-14.6-3.2-5.3.0-0
+  ubi8-postgres-gis-14-31-tag: ubi8-14.6-3.1-5.3.0-0
+  ubi8-postgres-gis-component-15-33-tag: ubi8-15.1-3.3-0
+  ubi8-postgres-gis-component-14-33-tag: ubi8-14.6-3.3-2
+  ubi8-postgres-gis-component-14-32-tag: ubi8-14.6-3.2-2
+  ubi8-postgres-gis-component-14-31-tag: ubi8-14.6-3.1-2
+  ubi8-postgres-gis-component-13-31-tag: ubi8-13.9-3.1-2
+  ubi8-postgres-gis-component-13-30-tag: ubi8-13.9-3.0-2
+  ubi8-pgo-latest-image-tag: ubi8-5.3.0-0
+
+updates:
+- targetFile: .github/ISSUE_TEMPLATE/bug_report.md
+  template: hack/release-templates/github-ISSUE_TEMPLATE-bug-report.md
+  replacements:
+  - ubi8-pgo-latest-image-tag
+
+- targetFile: .github/ISSUE_TEMPLATE/feature_request.md
+  template: hack/release-templates/github-ISSUE_TEMPLATE-feature-request.md
+  replacements:
```

Comment:
:+1: I like that you made the values being replaced explicit for each file rather than a search across every file.

:question: That said, could we add more guidance on how to add new values? If I understand correctly, it would amount to adding the new variable (with the `{{value}}` format) in the file(s), then adding the new `release-tags` value along with updating an existing (or adding a new) entry under `updates`. I think either more information in the comment or possible a short README would suffice.